ui/test/message_popup/hint_card_test.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {mount, shallow} from 'enzyme';
+import {expect} from 'chai';
+import sinon from 'sinon';
+
+import RaisedButton from 'material-ui/RaisedButton';
+import HintCard from '../src/message_popup/hint_card.jsx';
```

Comment:
I think this import might fail because it would search for `src/message_popup/hint_card.jsx` inside of `ui/test`.

 Would this work instead?
```
../../src/message_popup/hint_card.jsx
```